src/Common/PrefixSubmitButton.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ipl\Web\Common;
+
+use ipl\Html\Attribute;
+use ipl\Html\Contract\FormSubmitElement;
+use ipl\Html\FormElement\SubmitElement;
+
+trait PrefixSubmitButton
+{
+    protected function createSubmitButton(FormSubmitElement $originalSubmitButton): SubmitElement
```

Comment:
I'm not sure the name of the trait and the method name properly represent their purpose.

For the trait name I think `DuplicateSubmitButton` suits. The same name should be used for the method, as that's what it does. Please document the method as well.

The term *prefix* is not relevant here, only in `CompatForm`.